dist/src/main/resources/wildfly/patches/standalone.xsl

```diff
@@ -349,7 +354,7 @@
                  numAvailSchedulerThreads="3">
         <xsl:attribute name="enabled">
           <xsl:choose>
-            <xsl:when test="$kettle.build.type='dev'">true</xsl:when>
+            <xsl:when test="$kettle.build.type='dev'">${hawkular.agent.enabled:true}</xsl:when>
```

Comment:
thanks for the merge
I think it's because the `$`, `{` and `}` aren't named entities in XML (but <, >, &, etc. are)